model/utils_test.go

```diff
@@ -99,15 +91,10 @@ func TestMapJson(t *testing.T) {
 	json := MapToJson(m)
 
 	rm := MapFromJson(strings.NewReader(json))
-
-	if rm["id"] != "test_id" {
-		t.Fatal("map should be valid")
-	}
-
+	assert.Equal(t, rm["id"], "test_id", "map should be valid")
+	
 	rm2 := MapFromJson(strings.NewReader(""))
-	if len(rm2) > 0 {
-		t.Fatal("make should be ivalid")
-	}
+	assert.LessOrEqual(t, len(rm2), 0, "make should be invalid")
```

Comment:
same here

Reply:
Same as above. `len(rm2)` is coming back ***as*** 0.
```
=== RUN   TestMapJson
--- FAIL: TestMapJson (0.00s)
    utils_test.go:97: 
        	Error Trace:	utils_test.go:97
        	Error:      	"0" is not less than "0"
        	Test:       	TestMapJson
        	Messages:   	make should be invalid
```